life.js

```diff
@@ -1,9 +1,10 @@
 const lifeBeyond = async () => {
   const character = await fetchCharacter();
-  const lifeDomain = character.classes.filter((characterClass) => characterClass.subclassDefinition.name == 'Life Domain')[0];
+  const lifeDomain = character.classes.filter((charClass) => charClass.subclassDefinition.name == 'Life Domain')[0];
   if (lifeDomain == null) {
     return;  // nothing to do here
   }
+  await new Promise(resolve => setTimeout(resolve, 3000));
```

Comment:
Right here is where the core of the problem occurs. 

I looked into what is happening as the dndbeyond application initializes, and from what I see they are no longer loading the page all at once. Now, they are loading individual components one at a time -- I assume to improve the perceived page-load times. 

Before the update, our request for character data would have resolved long after the sidebar was on the screen. But now, after we get the character data from the backend, we try to begin observing the sidebar in the DOM only to find there is no sidebar on the screen!

The setTimeout here addresses that timing problem by providing a little 3 second pause before moving on to attach the spell book observer.

🤔 However, I generally try to avoid using time-based delays (aka "sleeps") as they are not always dependable. While the sidebar may be added to one person's screen after 2 seconds, for another person it may be 4 seconds and they would see the same problem again.

So instead of a time-based delay, I will put together a commit to add an event-based delay which watches the `document.body` for the sidebar to appear before attaching the spell book observer.